src/replication.c

```diff
@@ -528,8 +825,12 @@ send_replica_handshake_query(replica_t *replica, spec_t *spec)
 	MTX_LOCK(&replica->r_mtx);
 	TAILQ_INSERT_TAIL(&replica->mgmt_cmd_queue, mgmt_cmd, mgmt_cmd_next);
 	MTX_UNLOCK(&replica->r_mtx);
+	if (write(replica->mgmt_eventfd1, &num, sizeof (num)) != sizeof (num)) {
+		REPLICA_NOTICELOG("Failed to inform to mgmt_eventfd for replica(%p)\n", replica);
```

Comment:
failed one need to be removed from linked list

Reply:
or let us leave it as it is now.. and also, lets return 0

Reply:
Freeing it now.

Reply:
hmm.. lets leave it as it is and return 0, because, epollout will come right after adding to epoll and that will take care of sending out the command even though this write is failed.